.gitignore

```diff
@@ -23,8 +23,9 @@ _testmain.go
 
 dist/builds/
 dist/release/
+.idea/
```

Comment:
You don't have to fix this, but you should put ignore files related to your dev environment in your global gitignore.

https://help.github.com/articles/ignoring-files/#create-a-global-gitignore